x-pack/plugin/mapper-unsigned-long/src/yamlRestTest/resources/rest-api-spec/test/80_ignore_malformed.yml

```diff
@@ -0,0 +1,186 @@
+---
+"unsigned_long with ignore_malformed in synthetic source":
+  - do:
+      indices.create:
+        index: test-synthetic
+        body:
+          mappings:
+            _source:
+              mode: synthetic
+            properties:
+              ul_ignored:
+                type: unsigned_long
+                ignore_malformed: true
+              ul_not_ignored:
+                type: unsigned_long
+                ignore_malformed: false
+
+  - do:
+      catch: bad_request
+      index:
+        refresh: true
+        index: test-synthetic
+        id: "1"
+        body: { "ul_ignored": 2000, "ul_not_ignored": { "key": "bar", value: 200 } }
+
+  - match: { error.root_cause.0.type: "document_parsing_exception" }
+
+  - do:
+      catch: bad_request
+      index:
+        refresh: true
+        index: test-synthetic
+        id: "2"
+        body: { "ul_ignored": { "key": "foo", "value": 100 }, "ul_not_ignored": 1000 }
+
+  - match: { error.root_cause.0.type: "document_parsing_exception" }
+
+  - do:
+      index:
+        refresh: true
+        index: test-synthetic
+        id: "3"
+        body: { "ul_ignored": [ 100, 200, 300 ], "ul_not_ignored": 3000 }
+
+  - do:
+      catch: bad_request
+      index:
+        refresh: true
+        index: test-synthetic
+        id: "4"
+        body: { "ul_ignored": [ { "key": "a", "value": 100 }, { "key": "b", "value": 200 } ], "ul_not_ignored": 4000 }
+
+  - match: { error.root_cause.0.type: "document_parsing_exception" }
+
+  - do:
+      catch: bad_request
+      index:
+        refresh: true
+        index: test-stored
+        id: "5"
+        body: { "ul_ignored": [1, { "key": "foo", "value": "bar" }, 3], "ul_not_ignored": 4000 }
```

Comment:
This is the case where I see different behaviour between stored and synthetic source.

Reply:
This in an operation against `test-stored` which does not exist at this time right. So this is going into dynamic fields code path.

Reply:
Yes I fixed it yesterday but didn't push since I was doing some more tests...that is the reason why I saw dynamic mapping kick in...